make.go

```diff
@@ -21,6 +22,24 @@ func commandOutput(name string, args ...string) string {
 	return strings.TrimSpace(string(out))
 }
 
+// List all the go files in the current directory, excluding this file
+func listFilesToBuild() []string {
+	goFiles := make([]string, 0)
+
+	files, err := ioutil.ReadDir(".")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, file := range files {
+		if strings.HasSuffix(file.Name(), ".go") && !strings.HasSuffix(file.Name(), "make.go") {
```

Comment:
Wondering if we should maybe consider `runtime.Caller()` instead of hardcoding "make.go", in case this filename ever changes.